docs/releases/5.0.txt

```diff
@@ -0,0 +1,354 @@
+============================================
+Django 5.0 release notes - UNDER DEVELOPMENT
+============================================
+
+*Expected December 2023*
+
+Welcome to Django 5.0!
+
+These release notes cover the :ref:`new features <whats-new-5.0>`, as well as
+some :ref:`backwards incompatible changes <backwards-incompatible-5.0>` you'll
+want to be aware of when upgrading from Django 4.2 or earlier. We've
+:ref:`begun the deprecation process for some features
+<deprecated-features-5.0>`.
+
+See the :doc:`/howto/upgrade-version` guide if you're updating an existing
+project.
+
+Python compatibility
+====================
+
+Django 5.0 supports Python 3.10, 3.11, and 3.12. We **highly recommend** and
+only officially support the latest release of each series.
+
+Third-party library support for older version of Django
+=======================================================
+
+Following the release of Django 5.0, we suggest that third-party app authors
+drop support for all versions of Django prior to 4.2. At that time, you should
+be able to run your package's tests using ``python -Wd`` so that deprecation
+warnings appear. After making the deprecation warning fixes, your app should be
+compatible with Django 5.0.
+
+.. _whats-new-5.0:
+
+What's new in Django 5.0
+========================
+
+Minor features
+--------------
+
+:mod:`django.contrib.admin`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* ...
+
+:mod:`django.contrib.admindocs`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The default iteration count for the PBKDF2 password hasher is increased from
```

Comment:
This belongs in the auth section, not admindocs. ;-)

Reply:
Thanks!
- #16500